ethcore/src/engines/clique/mod.rs

```diff
@@ -744,14 +756,6 @@ impl Engine<EthereumMachine> for Clique {
 		}
 	}
 
-	fn stop(&mut self) {
```

Comment:
Do we actually need `stop` on the `Engine` trait then? Is it used anywhere?

Reply:
> Can you change https://github.com/paritytech/parity-ethereum/blob/master/ethcore/src/engines/mod.rs#L429 to fn stop(&self) it is broken, because Engine is stored as Arc<Engine> and doesn't support mutation.

It is not used and it is broken. Either remove it or change it to `fn stop(&self)`

Reply:
Yes, that's planned for the follow up PR. :)

(What, you a mind reader @tomusdrw ???)